### PR TITLE
HOP-66

### PR DESCRIPTION
**Pull request by nadment**
HOP-66 and assembly plugin database ID

### CODE REVIEW

File: assemblies/plugins/databases/as400-assemblies/pom.xml [hunk 1]
```diff
@@ -18,13 +18,21 @@
     <name>Hop Assemblies Plugins Databases AS400</name>
     <description></description>
 
+    <properties>
+        <jt400.version>6.1</jt400.version>
```
Comment (hansva) [inline assemblies/plugins/databases/as400-assemblies/pom.xml]: Maybe bump to 10.2

File: assemblies/plugins/databases/firebird-assemblies/pom.xml [hunk 2]
```diff
@@ -10,21 +10,34 @@
...
+      		<groupId>org.firebirdsql.jdbc</groupId>
+      		<artifactId>jaybird</artifactId>
+      		<version>${jaybird.version}</version>
+      		<exclusions>
+       			<exclusion>
+       				<groupId>*</groupId>
+       				<artifactId>*</artifactId>
+      			</exclusion>
+      		</exclusions>
+    	</dependency>
```
Comment (hansva) [inline assemblies/plugins/databases/firebird-assemblies/pom.xml]: We can't include the jaybird driver it is licensed under GPL 2.1 and is incompatible with Apache 2

File: assemblies/plugins/databases/sapdb-assemblies/pom.xml [hunk 3]
```diff
@@ -18,13 +18,27 @@
...
+      		<groupId>com.sap</groupId>
+      		<artifactId>sapdbc</artifactId>
+      		<version>${sapdbc.version}</version>
+      		<exclusions>
+        		<exclusion>
+          			<groupId>*</groupId>
+          			<artifactId>*</artifactId>
+        		</exclusion>
+      		</exclusions>
+    	</dependency>
```
Comment (hansva) [inline assemblies/plugins/databases/sapdb-assemblies/pom.xml]: I can't find what license this driver is created under, do you happen to know which it is?
If we can't find it we will have to remove this include top avoid future issues

Comment (nadment) [reply assemblies/plugins/databases/sapdb-assemblies/pom.xml]: License not found, except that: "In October 2007 SAP assumed full sales and commercial support for MaxDB. MaxDB 7.6 is now closed source, available free-of-charge (without support, and with usage restrictions) for use with non-SAP applications."

**Pull request merged**